terragrunt/modules/sync-team/confirmation.tf

```diff
@@ -0,0 +1,132 @@
+locals {
+  domain = "confirmation.sync-team-prod.rust-lang.net"
+}
+
+data "aws_ssm_parameter" "confirmation_parameters" {
+  for_each = toset([
+    "/prod/sync-team-confirmation/github-oauth-client-id",
+    "/prod/sync-team-confirmation/github-oauth-client-secret",
```

Comment:
Hard coding these and the domain seem fine, but we may want to consider passing them as variables.